Assets/Scripts/WorldEngine/World.cs

```diff
@@ -134,11 +134,11 @@ public static class RngOffsets
 
 public enum GenerationType
 {
-    Temperature = 0x01,
+    Temperature = 0x03, // also generate rainfall
     Rainfall = 0x02,
-    TerrainNormal = 0x07,
-    TerrainRegeneration = 0x0B,
-    TemperatureRegeneration = 0x10,
+    TerrainNormal = 0x07, // generate altitude, temperature, and rainfall
+    TerrainRegeneration = 0x0B, // regenerate altitude, generate temperature and rainfall
+    TemperatureRegeneration = 0x12, // also generate rainfall
     RainfallRegeneration = 0x20,
     LayerRegeneration = 0x40
 }
```

Comment:
Any reason to change the numbers? I'm just curious.
Also ty for comments

Reply:
So, previously, when you used one of the editor sliders to change the global temperature or rainfall, the editor would regenerate just the temperature or the rainfall accordingly since those two parameters where independent of each other (but not from altitude). But, recently I made a change that makes rainfall slightly dependent of temperature (less temperature = more rainfall). Thus, changing the global temperature levels should also affect the global rainfall levels. This change makes it so that enabling the "temp regen" flag will also enable the "rainfall regen" flag